docs/script.js

```diff
@@ -1,14 +1,112 @@
 // Tabs
-const tabButtons = document.querySelectorAll("[role=tab]");
+// Tabs > Sample Tabs
+const tabList = document.querySelector("[aria-label='Sample Tabs']");
+const tabButtons = tabList.querySelectorAll("[role=tab]");
 tabButtons.forEach((tabButton) => {
-  tabButton.addEventListener("click", (e) => {
+  tabButton.addEventListener("mousedown", (e) => {
     e.preventDefault();
     const tabContainer = e.target.parentElement.parentElement;
     const targetId = e.target.getAttribute("aria-controls");
     tabButtons.forEach((_tabButton) =>
       _tabButton.setAttribute("aria-selected", false)
     );
     tabButton.setAttribute("aria-selected", true);
+    tabButton.focus();
+    tabContainer
+      .querySelectorAll("[role=tabpanel]")
+      .forEach((tabPanel) => tabPanel.setAttribute("hidden", true));
+    tabContainer
+      .querySelector(`[role=tabpanel]#${targetId}`)
+      .removeAttribute("hidden");
+  });
+});
+tabButtons.forEach((tabButton) => {
+  tabButton.addEventListener("focus", (e) => {
+    e.preventDefault();
+    const tabContainer = e.target.parentElement.parentElement;
+    const targetId = e.target.getAttribute("aria-controls");
+    tabButtons.forEach((_tabButton) =>
+      _tabButton.setAttribute("aria-selected", false)
+    );
+    tabButton.setAttribute("aria-selected", true);
+    tabContainer
+      .querySelectorAll("[role=tabpanel]")
+      .forEach((tabPanel) => tabPanel.setAttribute("hidden", true));
+    tabContainer
+      .querySelector(`[role=tabpanel]#${targetId}`)
+      .removeAttribute("hidden");
+  });
+});
+
+// Tabs > Tabs Template
+const tabList_TabsTemp = document.querySelector("[aria-label='Tabs Template']");
```

Comment:
- Sure thing, I'll change that. By the way, have you ever considered writing coding style guidelines? It'll be helpful for future contributors to know which coding styles to follow.
- As I've stated before, Javascript is not my area of expertise, so yeah, I did notice that I did a lot of code repetition 😅. I'll see what I can do.

Reply:
I actually never considered putting up any contributing guidelines since it's just a fun project, I don't wanna spend the extra effort making up "rules" that people would never bother reading and are forced to follow.

With that said, don't stress about complying with what I suggested, these are just minor points, since you have spent the time contributing, I just feel I need to review and provide proper feedback so you may see some value in your contribution in return. **Consistency** and **DRY** (Don't Repeat Yourself) are prolly the most fundamental standards in any programming language, not just JavaScript.